manim/utils/module_ops.py

```diff
@@ -115,6 +115,9 @@ def prompt_user_for_choice(scene_classes):
         sys.exit(2)
     except EOFError:
         sys.exit(1)
+    except ValueError:
+        print("No scenes chosen. Exiting")
+        sys.exit(0)
```

Comment:
IMO, it's an error so it should exit with status 1
```suggestion
        sys.exit(1)
```